### PR TITLE
Update grammar and style

### PR DESCRIPTION
**Pull request by spethso**
Update grammar and style for the texts according to Grammarly Premium.

### CODE REVIEW

**Comment by spethso** [comment]
This PR updates the grammar and style of the main texts of our web page. Major rewrites were to obtain conciseness, e.g. rewriting unclear passive voice to more clear active voice.

Please check that:

- [ ] No major texts were forgotten to review.
- [x] No HTML tags were accidentally deleted.

File: static/index.html [hunk 1]
```diff
@@ -66,7 +66,7 @@ <h1>Welcome to the Microsoft Campus Community</h1>
 
             <section id="bridgeSection">
                 <div class="sectionDiv">
-                    <p>It is not all about coding. We want you to connect with peers and industry experts. Foster your own network, learn new technologies and accelerate your personal growth. To be a future leader in tech you should be open to share expertise as well as experience and inspire others to live up to their potential.</p>
+                    <p>It is not all about coding. We want you to connect with peers and industry experts. Foster, your network, learn new technologies and accelerate your personal growth. To be a future leader in tech, you should be open to share expertise as well as experience and inspire others to live up to their potential.</p>
```
Comment (MalteRei) [inline static/index.html]: Is it Foster your network, learn new technologies or Foster, your network, learn new technologies?

**Review by MalteRei — APPROVED** [review]
Looks good. I am not sure about the comma after foster

File: static/index.html (hunk 1 above)
Comment (spethso) [reply static/index.html]: I think the comma is there by mistake. The original one sounds better, here.

**Review by MalteRei — APPROVED** [review]

**Comment by MalteRei** [comment]
Do you also want to review 
https://campus-community.org/legal.html
and
https://campus-community.org/privacy.html
with Grammarly? They are copied from somewhere. However, I already found a formatting issue.

**Comment by spethso** [comment]
> Do you also want to review
> https://campus-community.org/legal.html
> and
> https://campus-community.org/privacy.html
> with Grammarly? They are copied from somewhere. However, I already found a formatting issue.

I just wanted to review the texts. However, since they are written in German, Grammarly is not able to review them.

**Pull request merged**